src/app/shared-services/timetable/timetable.service.spec.fixtures.ts

```diff
@@ -0,0 +1,95 @@
+import { Inference } from 'src/app/model/inference'
+import { InferenceType } from '../inferences/engine/types'
+
+export const oneMondayFiveToNine = new Inference(
+  'inferenceId',
```

Comment:
The IDs `inferenceId` and `inferenceId2` are used (at least) twice each - is this intentional?

Reply:
The way the tests are set up right now only inferences with different IDs are passed to the timetable together, so it shouldn't make a difference.